examples/topics/Seattle_Lidar.ipynb

```diff
@@ -0,0 +1,449 @@
+    "This notebook uses datashader to visualize Lidar elevation data from [the Puget Sound Lidar consortium](http://pugetsoundlidar.ess.washington.edu/), a source of Lidar data for the Puget Sound region of Washington, U.S."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Lidar Elevation Data\n",
+    "\n",
+    "Example X,Y,Z scattered point elevation data from the unpacked 7zip files (unpacked as .gnd files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "! head ../data/q47122d2101.gnd\n",
+    "```\n",
+    "```\n",
+    "X,Y,Z\n",
+    "1291149.60,181033.64,467.95\n",
+    "1291113.29,181032.53,460.24\n",
+    "1291065.38,181035.74,451.41\n",
+    "1291113.16,181037.32,455.51\n",
+    "1291116.68,181037.42,456.20\n",
+    "1291162.42,181038.90,467.81\n",
+    "1291111.90,181038.15,454.89\n",
+    "1291066.62,181036.73,451.41\n",
+    "1291019.10,181035.20,451.64\n",
+    "```\n",
+    "\n",
+    "The Seattle area example below loads 25 `.gnd` elevation files like the one above. We'll download, cache and read the data using `intake`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import intake\n",
+    "\n",
+    "cat = intake.open_catalog('../catalog.yml')\n",
+    "list(cat)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lidar = cat.seattle_lidar()\n",
+    "df = lidar.to_dask()\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Geographic Metadata\n",
+    "\n",
+    "Since the data are geographic, we need to know the coordinate reference system (CRS) of the X and Y. All the geographic metadata is stored in the data source entry in the intake catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyproj import Proj, transform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lidar.metadata['crs']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "washington_state_plane = Proj(init='epsg:2855')   # Washington State Plane North EPSG code\n",
+    "web_mercator = Proj(init='epsg:3857')             # Mercator projection EPSG code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "FT_2_M = 0.3048  \n",
+    "\n",
+    "def convert_coords(ddf):\n",
+    "    lon, lat = transform(washington_state_plane, web_mercator, ddf.X.values * FT_2_M, ddf.Y.values * FT_2_M)\n",
+    "    ddf['meterswest'], ddf['metersnorth'] = lon, lat \n",
+    "    return ddf[['meterswest', 'metersnorth', 'Z']]"
```

Comment:
If this function is specific to Dask dataframes, please add a docstring saying so.  Otherwise, I'd prefer just `df` to `ddf`.